### DISCUSSION TITLE
How to kill Mac Mouse Fix via terminal [enhancement]

### DISCUSSION THREAD

**Discussion by lywlywly** [noah-nuebling/mac-mouse-fix#898] [Questions] [answered]
## Description
Many other 3rd party tools on macOS have a daemon process running in the background, while it seems that Mac Mouse Fix does not. Is there a way to stop Mac Mouse Fix via terminal?

## Rationale
I experienced strange behavior of mouse freezing and I want to check if it is the problem of Mac Mouse Fix. I was still able to use keyboard, but I cannot find the process for Mac Mouse Fix background service.

## Additional info
–

**Comment by noah-nuebling** [comment, marked as answer]
Hi, thanks for your feedback!

When Mac Mouse Fix is enabled, there is a process running in the background that interacts with your mouse. It is called "Mac Mouse Fix Helper".

When you kill this process via Activity Monitor or the Terminal, it is restarted automatically because it is kept alive by the macOS.

To kill this process and prevent it from being restarted, try this command:

```
launchctl remove com.nuebling.mac-mouse-fix.helper
```

**Reply by lywlywly** [reply]
Thanks.
